### DISCUSSION TITLE
What kind of animal is a narwhal?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#883] [Galaxy Trivia] [answered]
**Bonus:** How long could their tusk be?

**Comment by peterbaccaro** [comment, marked as answer]
The narwhal is an **odontocete** or **toothed whale**, but is different from all other toothed whales in that it has no teeth in its mouth. Instead, male narwhals have a single long, straight tooth (or tusk) that protrudes two to three meters out of the upper left jaw.

A narwhal tusk, which is actually a specialized tooth, can grow up to **10 feet (3 meters)** in length. However, the average length of a narwhal tusk is typically around 6 to 9 feet (1.8 to 2.7 meters).
